applications/asset_tracker_v2/src/main.c

```diff
@@ -33,7 +34,7 @@
 #include <logging/log.h>
 #include <logging/log_ctrl.h>
 
-LOG_MODULE_REGISTER(MODULE, CONFIG_APPLICATION_MODULE_LOG_LEVEL);
+LOG_MODULE_REGISTER(app_test_placeholder, CONFIG_APPLICATION_MODULE_LOG_LEVEL);
```

Comment:
I have no idea about testing, but I am surprised to see a test placeholder being used in the main application file.

Reply:
The changes to `main.c` is just a workaround needed in order to properly run the app in qemu. The root cause will be fixed with https://github.com/nrfconnect/sdk-nrf/pull/7147.